test/REGRESSIONS

```diff
@@ -686,6 +679,668 @@ Reviewed 2015-01-23
+cray-module-correctness-cray-xc-slurm-none-gnu
+Inherits 'cray-module-correctness-cray-xc-none-gnu'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===================================================
+
+
+============================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-cray
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-cray'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+============================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-pgi
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-pgi'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+=============================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-intel
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-intel'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=============================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-aries-gnu
+Inherits 'cray-module-correctness-cray-xc-gasnet-aries-gnu'
+and ''cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+==========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-cray
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-cray'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+==========================================================
+
+
+=========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-pgi
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-pgi'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=========================================================
+
+
+===========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-intel
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-intel'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+===========================================================
+
+
+=========================================================
+cray-module-correctness-cray-xc-slurm-gasnet-mpi-gnu
+Inherits 'cray-module-correctness-cray-xc-gasnet-mpi-gnu'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=========================================================
+
+
+=====================================================
+cray-module-correctness-cray-xc-slurm-ugni-intel
+Inherits 'cray-module-correctness-cray-xc-ugni-intel'
+and 'cray-module-correctness-cray-xc-slurm'
+Reviewed 2015-01-29
+=====================================================
+
```

Comment:
I'm a little concerned by the huge number of blank categories here and would like to strategize about better ways to organize that didn't result in quite so many cases.  In particular, for whitebox testing, the choice to not go beyond certain levels of wildcards by default was made for exactly this reason -- to not have a significant number of empty categories.  Do we really believe that this listing is the optimal organization?  Again, I'll be curious what @thomasvandoren thinks here.